piet/src/lib.rs

```diff
@@ -4,13 +4,17 @@
 //! graphics API, in the tradition of [`PostScript`]. It is built on top of
 //! [`kurbo`], a 2D geometry library.
 //!
-//! The main interface is the [`RenderContext`] trait.
+//! The main interface is the [`RenderContext`] trait. Another useful place
+//! to look at if you're getting started would be the `Device` and
+//! `BitmapTarget` documentation on [`piet-common`], to create `RenderContext`
+//! objects and extract rendered pixels from them.
```

Comment:
I'd discard this in favour of something like the edit below:

Reply:
Was it supposed to have a code suggestion on this comment too? I already applied your other one, otherwise, I don't follow

Reply:
I don't think there's an easy way for me to 'suggest' discarding an edit? I guess I can just suggest an edit reverting to the previous text 🤔

```suggestion
//! The main interface is the [`RenderContext`] trait.
```